### PR TITLE
RELATED: BB-534 everywhere in table display dash instead of empty value

### PR DESCRIPTION
**Pull request by tomassatinsky**
Empty value occurs when data value is null and the metric doesn't have custom formatting for null

I've also put some nulls to the fixtures so we make sure in tests and storybook that also this value is being handled properly

### CODE REVIEW

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/550/ : SUCCESS in 21s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/347/ : FAILURE in 5m 44s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/119/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/551/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/275/ : UNSTABLE in 6m 13s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/120/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/553/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/276/ : UNSTABLE in 4m 17s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/552/ : SUCCESS in 21s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/348/ : FAILURE in 5m 08s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/554/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/349/ : FAILURE in 3m 13s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/555/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/277/ : FAILURE in 4m 00s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/557/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/278/ : FAILURE in 3m 25s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/556/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/350/ : SUCCESS in 3m 45s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/558/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/279/ : FAILURE in 3m 09s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/124/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/559/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/280/ : UNSTABLE in 4m 54s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/560/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/351/ : SUCCESS in 5m 02s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/125/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/561/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/281/ : SUCCESS in 5m 12s

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by tomassatinsky** [comment]
extended test - storybook

File: src/Table/utils/cell.js [hunk 1]
```diff
@@ -17,15 +17,27 @@ export function getCellClassNames(rowIndex, columnKey, isDrillable) {
...
-    const { label, color } = colors2Object(formattedNumber);
-    const style = color ? { color } : {};
+    const { label: origLabel, color } = colors2Object(formattedNumber);
+
+    let style;
+    let label;
+    if (origLabel === '') {
+        label = '–';
+        style = {
+            color: '#94a1ad',
```
[the remark this answers is not in the document]
Comment (tomassatinsky) [reply src/Table/utils/cell.js]: Good point. Can you have a look now?

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/563/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/354/ : FAILURE in 1m 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/565/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/355/ : SUCCESS in 1m 39s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/127/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/566/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/283/ : SUCCESS in 4m 05s

File: src/Table/utils/cell.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (xMort) [reply src/Table/utils/cell.js]: It looks better now 👍

**Comment by BugsBunny338** [comment]
lgtm

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-merge-compliance-check/107/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/356/ : SUCCESS in 1m 38s

**Pull request merged**